src/renderer/components/dex/OrderBook.vue

```diff
@@ -32,7 +32,7 @@
             <div class="body">
               <div class="row" v-for="(bid, index) in $store.state.orderBook.bids" :key="index">
                 <div class="cell price green" @click="setPrice(bid.price)">{{ $formatNumber(bid.price) }}</div>
-                <div class="cell quantity" @click="setQuantity(bid.quantity)">{{ $formatNumber(bid.quantity) }}</div>
+                <div class="cell quantity" @click="setQuantity(index, $store.state.orderBook.bids)">{{ $formatNumber(bid.quantity) }}</div>
```

Comment:
You don't need to pass `$store.state.orderBook.bids` as it is always the same. Just access it in the method.

Reply:
I do - because the method doesn't know which side I am targeting. How would I know to use either the bid or ask?

Reply:
I pass in the list knowing it is either can be the bid or ask list. It doesn't matter which. It allows for this pattern.

Reply:
Please see comment. This is passed so I can know which side to target (ask/bid)

Reply:
I think the only way to avoid it would be if he had separate methods for setting buy and sell quantity, which isn't necessary; so I approved.